open_feature/open_feature_client.py

```diff
@@ -359,3 +362,24 @@ def _create_provider_evaluation(
             raise TypeMismatchError()
 
         return value
+
+    def __extract_evaluation_options(
```

Comment:
I'm not sure if perhaps your comment on the thread beneath this one affects this, however, I'd question why this method is prefixed with a double underscore? I think you are intending to just mark this method as private (which should use a single leading underscore by convention) rather than utilise any name mangling functionality (which uses the double leading underscore).

Reply:
Yeah this would be removed entirely, but yes it's mostly meant to be private not name mangled.